python_modules/dagster/dagster/_core/storage/dagster_run.py

```diff
@@ -541,6 +541,7 @@ class RunsFilter(IHaveNew, LegacyNamedTupleMixin):
     updated_before: Optional[datetime]
     created_after: Optional[datetime]
     created_before: Optional[datetime]
+    exclude_subruns: Optional[bool]
```

Comment:
I went with an API here that would make getting this PR up for discussion faster, but there are some other options for what we could do here:
-  "anti-tag" scheme where if the key, value pair is present on the run we exclude it. 
- similar to above, but where just the presence of the key excludes the run

Reply:
1 makes the most sense as a standalone feature

Reply:
if we go with 1, the function to fetch the runs for the runs feed would just create an "anti tag" for each of the backfills we already fetched

Reply:
Can we mark this as experimental?

Are we set on subruns?

Did we consider `exclude_backfills` / `exclude_backfill_runs`?

Reply:
not set on subruns. I like `exclude_backfill_runs` better than `exclude_backfills` since `exclude_backfills` sounds like we're excluding the backfill objects themselves

Reply:
Can we switch to `exclude_backfill_runs`?  Are we committed to using `subruns` in the UI?